qiskit_experiments/framework/base_experiment.py

```diff
@@ -373,9 +378,8 @@ def _transpiled_circuits(self) -> List[QuantumCircuit]:
 
         This function can be overridden to define custom transpilation.
         """
-        transpile_opts = copy.copy(self.transpile_options.__dict__)
-        transpile_opts["initial_layout"] = list(self.physical_qubits)
-        transpiled = transpile(self.circuits(), self.backend, **transpile_opts)
+        circuits = [map_qubits(c, self.physical_qubits) for c in self.circuits()]
```

Comment:
Why you don't take actual qubit size from the backend when available?

Reply:
Just because I had never seen it matter that the circuit size matched the backend. We can set it if it matters (for qasm3?).